ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fileinput/v2/fileinput/clientlibs/editor/js/editDialog.js

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * Copyright 2023 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+(function($) {
+    "use strict";
```

Comment:
This should also be re-used from v1 and not copy pasted, you can directly refer the v1 client library in the v2 dialog content XML as extra client libs.

Reply:
same that discussed this with Sakshi and we concluded the discussion with making V2 as totally separate with no dependency on V1.

Reply:
Reused from V1 only removed now.